plugins/inputs/win_perf_counters/win_perf_counters_test.go

```diff
@@ -12,7 +12,6 @@ import (
 )
 
 func TestWinPerfcountersConfigGet1(t *testing.T) {
-	validmetrics := itemList{}
```

Comment:
itemList was removed in this pull request https://github.com/influxdata/telegraf/pull/2800